Description: Set default job error policy to "Retry", since it is less confusing and a better default on desktop machines. This patch is only applied when building on Ubuntu.
Author: Martin Pitt <martin.pitt@ubuntu.com>

Index: cups-1.4.6/scheduler/conf.c
===================================================================
--- cups-1.4.6.orig/scheduler/conf.c	2011-07-14 13:03:03.479773075 +0200
+++ cups-1.4.6/scheduler/conf.c	2011-07-14 13:03:08.829773167 +0200
@@ -653,7 +653,7 @@
   cupsdClearString(&BrowseLocalOptions);
   cupsdClearString(&BrowseRemoteOptions);
 
-  cupsdSetString(&ErrorPolicy, "stop-printer");
+  cupsdSetString(&ErrorPolicy, "retry-job");
 
 #ifdef HAVE_LDAP
   cupsdClearString(&BrowseLDAPBindDN);
